frontend/views/containers/chatroom/SendArea.vue

```diff
@@ -397,7 +396,7 @@ export default ({
     },
     startMention (keyword, position) {
       const all = makeMentionFromUsername('').all.slice(1)
-      const availableMentions = this.isPrivateDirectMessage()
+      const availableMentions = this.users.length === 2
```

Comment:
What is this `this.users.length === 2` check doing?

Should it be `>` ?

If not, can you add a comment explaining what's going on here?

Reply:
Nice catch. It should be `<=2` here.
For better understanding, I will update it.
```
let availableMentions = this.users
// NOTE: '@all' mention should only be needed when the members are more than 3
if (this.users.length > 2) {
  availableMentions = [
    ...availableMentions,
    {
      username: all,
      displayName: all,
      picture: '/assets/images/horn.png'
    }
  ]
}
```